protocol/x/clob/keeper/liquidations.go

```diff
@@ -452,6 +452,28 @@ func (k Keeper) GetFillablePrice(
 		return nil, err
 	}
 
+	// stat liquidation order for negative TNC when in DeliverTx
+	if tncBig.Sign() < 0 && !ctx.IsCheckTx() && !ctx.IsReCheckTx() {
```

Comment:
Why don't we want to also stat this in `PrepareCheckState`?

Wouldn't it help determine whether some liq daemons are seeing negative TNC subaccounts in a block?

Reply:
My reasoning was that this function can be called twice per liquidation in PrepareCheckState which would make the metric confusing and difficult to interpret. I don't think we'd be able to meaningfully compare the PrepareCheckState stats with the DeliverTx stats

Reply:
What happens if there are negative TNC subaccounts but they can't be liquidated, and insurance fund balance is too high to deleverage them? Wouldn't we want to know how many negative TNC subaccounts existed if that was the case?

Can we just return the subaccount TNC from `GetFillablePrice` as well? Would that allow us to stat this in `PrepareCheckState`?